### PR TITLE
Making pivy-impoter build fat jar by default, and updating docs.

### LINKED ISSUE #47: how to include custom ivy-repo generated by pivy-importer in build.gradle
Using the ivy repo in my project's gradle file in the following way gives an error

``` 
apply plugin: "com.linkedin.python-sdist"
    repositories {
        ivy {url '<DIR>/pygradle/build/ivy-repo/'}
    }
```

```
> Could not find pypi:requests:2.9.1.
  Searched in the following locations:
      file:<DIR>/pygradle/build/ivy-repo/pypi/requests/2.9.1/ivy-2.9.1.xml
      file:<DIR>/pygradle/build/ivy-repo/pypi/requests/2.9.1/requests-2.9.1.jar
  Required by:
 <user>
> Could not find pypi:numpy:1.9.0.
  Searched in the following locations:
      file:<DIR>/pygradle/build/ivy-repo/pypi/numpy/1.9.0/ivy-1.9.0.xml
      file:<DIR>/pygradle/build/ivy-repo/pypi/numpy/1.9.0/numpy-1.9.0.jar
  Required by:
 <user>

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED
```

Why is it looking for .jar and .xml files? The `ivy-repo` has `.ivy` and `.tar.gz` files.

(Sorry for so many questions. If you would rather have me post on stackoverflow, please let me know the tag I should use.)

### PR DESCRIPTION
**Pull request by ethankhall**
Fixes #47 #45

### CODE REVIEW

File: docs/pivy-importer.md [hunk 1]
```diff
@@ -16,3 +33,19 @@ To enable this feature you need to add the option `--replace` followed by a list
...
 replacements by joining them with a comma.
  
+## Usage in Gradle
+To be able to consume the artifacts produced by the pivy-importer, you'll have to define a [repository](https://docs.gradle.org/current/userguide/dependency_management.html#sec:repositories) in Gradle. We use a specific format for the repository that you'll have to define. Here is an example definition in Gradle. You would add this to your project's build.gradle.
+
+```
+repositories {
+  ivy {
+    name 'pypi-local'   //optional, but nice
+    url "/path/to/repo'
```
Comment (sonalgupta) [inline docs/pivy-importer.md]: the quotation marks don't match

File: docs/pivy-importer.md [hunk 2]
```diff
@@ -2,10 +2,27 @@
...
 several features that you may find useful in dealing with pypi projects.
+
+## Getting the Jar
+
+### Published Jar
+
+We publish the importer into bintray. You can find all the copies at https://bintray.com/linkedin/maven/pivy-importer.
+
+At the time of writing this doc, the current version is 0.3.16, so the jar you would want to download is
+https://dl.bintray.com/linkedin/maven/com/linkedin/pygradle/pivy-importer/0.3.16/:pivy-importer-0.3.16-all.jar .
```
Comment (sonalgupta) [inline docs/pivy-importer.md]: typo in the URL. There is no ":" before the name of the jar file.

File: docs/pivy-importer.md [hunk 3]
```diff
@@ -2,10 +2,27 @@
...
+
+To build the pivy-importer you'll need to run `./gradlew build` in the pivy-importer package. This will product several
+jars in `build/pivy-importer/libs/`. We product a normal jar that could be used when consuming in other projects and a 
+fat jar, with all the dependencies added into a single executable. This one is the one you would use on the commandline.
+It follows the naming convention `pivy-importer-<VERSION>-all.jar`.
  
 ## General Usage
 
-`java -jar pivy-importer.jar --repo /path/to/destination virtualenv:15.0.1 pip:7.1.2`
+`java -jar pivy-importer-VERSION-all.jar --repo /path/to/destination virtualenv:15.0.1 pip:7.1.2`
```
Comment (sonalgupta) [inline docs/pivy-importer.md]: will be great to have a full runnable example here. This command right now gives error about `Unable to find version 0.7 for alabaster` and `Unable to find version 0a for pytz`; that is, all the libs that require `--replace`.

File: docs/pivy-importer.md (hunk 1 above)
Comment (ethankhall) [inline docs/pivy-importer.md]: Strange... I copied this from the integ tests setup. Nice catch.

File: docs/pivy-importer.md [hunk 4]
```diff
@@ -2,10 +2,27 @@
...
+We publish the importer into bintray. You can find all the copies at https://bintray.com/linkedin/maven/pivy-importer.
+
+At the time of writing this doc, the current version is 0.3.16, so the jar you would want to download is
+https://dl.bintray.com/linkedin/maven/com/linkedin/pygradle/pivy-importer/0.3.16/:pivy-importer-0.3.16-all.jar .
+This artifact has all dependencies bundled together to make a nice CLI. 
+
+### Building it Locally
+
+To build the pivy-importer you'll need to run `./gradlew build` in the pivy-importer package. This will product several
+jars in `build/pivy-importer/libs/`. We product a normal jar that could be used when consuming in other projects and a
```
Comment (sholsapp) [inline docs/pivy-importer.md]: > We product a normal jar

Typo.

File: docs/pivy-importer.md [hunk 5]
```diff
@@ -2,10 +2,27 @@
...
+
+To build the pivy-importer you'll need to run `./gradlew build` in the pivy-importer package. This will product several
+jars in `build/pivy-importer/libs/`. We produce a normal jar that could be used when consuming in other projects and a 
+fat jar, with all the dependencies added into a single executable. This one is the one you would use on the commandline.
+It follows the naming convention `pivy-importer-<VERSION>-all.jar`.
  
 ## General Usage
 
-`java -jar pivy-importer.jar --repo /path/to/destination virtualenv:15.0.1 pip:7.1.2`
+`java -jar pivy-importer-VERSION-all.jar --repo /path/to/destination --replace alabaster:0.7=alabaster:0.7.1,pytz:0a=pytz:2016.4virtualenv:15.0.1 pip:7.1.2`
```
Comment (sonalgupta) [inline docs/pivy-importer.md]: a space is missing before "virtualenv"

File: docs/pivy-importer.md [hunk 6]
```diff
@@ -2,10 +2,27 @@
...
+
+We publish the importer into bintray. You can find all the copies at https://bintray.com/linkedin/maven/pivy-importer.
+
+At the time of writing this doc, the current version is 0.3.16, so the jar you would want to download is
+https://dl.bintray.com/linkedin/maven/com/linkedin/pygradle/pivy-importer/0.3.16/pivy-importer-0.3.16-all.jar .
+This artifact has all dependencies bundled together to make a nice CLI. 
+
+### Building it Locally
+
+To build the pivy-importer you'll need to run `./gradlew build` in the pivy-importer package. This will product several
```
Comment (zvezdan) [inline docs/pivy-importer.md]: s/This will product/This will produce/

File: docs/pivy-importer.md [hunk 7]
```diff
@@ -16,3 +33,98 @@ To enable this feature you need to add the option `--replace` followed by a list
 oldName:oldVersion=newName:newVersion (`alabaster:0.7=alabaster:0.7.1`). You can provide multiple 
 replacements by joining them with a comma.
  
+## Usage in Gradle
+To be able to consume the artifacts produced by the pivy-importer, you'll have to define a [repository](https://docs.gradle.org/current/userguide/dependency_management.html#sec:repositories) in Gradle. We use a specific format for the repository that you'll have to define. Here is an example definition in Gradle. You would add this to your project's build.gradle.
```
Comment (zvezdan) [inline docs/pivy-importer.md]: Add blank line after the title.

**Pull request merged**